lib/spack/spack/relocate.py

```diff
@@ -651,8 +651,7 @@ def is_relocatable(spec):
     return True
 
 
-def file_is_relocatable(
-        file, paths_to_relocate=[spack.store.layout.root, spack.paths.prefix]):
```

Comment:
What happens if you turn this into a tuple instead? Not having lists as defaults for function arguments is a Python's gotcha, so I am wondering if this is the real issue here.

Reply:
Just checked. Apparently that's not the issue here.